Scrabble.java

```diff
@@ -48,24 +51,43 @@ public static void init() {
 
 	// Checks if the given word is in the dictionary.
 	public static boolean isWordInDictionary(String word) {
-		//// Replace the following statement with your code
+		for(int i = 0;i < DICTIONARY.length;i++){
+			if (word == DICTIONARY[i]) {
+				return true;
+			}
+		}
 		return false;
 	}
 	
 	// Returns the Scrabble score of the given word.
 	// If the length of the word equals the length of the hand, adds 50 points to the score.
 	// If the word includes the sequence "runi", adds 1000 points to the game.
 	public static int wordScore(String word) {
-		//// Replace the following statement with your code
-		return 0;
+		int score = 0;
+		word = word.toLowerCase();
+		for(int i = 0;i < word.length();i++){
+			if (word != "") {

```

Comment:
using == and != on strings is inconsistent in Java, use `str.equals("")`, or `str.length() == 0 `to check for empty string